homeassistant/components/rako/light.py

```diff
@@ -0,0 +1,203 @@
+"""Platform for light integration."""
+from __future__ import annotations
+
+import asyncio
+import logging
+from typing import TYPE_CHECKING, Any, Callable
+
+import python_rako
+from python_rako.exceptions import RakoBridgeError
+from python_rako.helpers import convert_to_brightness, convert_to_scene
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS,
+    SUPPORT_BRIGHTNESS,
+    LightEntity,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity import DeviceInfo, Entity
+
+from .const import DOMAIN
+from .util import create_unique_id
+
+if TYPE_CHECKING:
+    from . import RakoDomainEntryData
+    from .bridge import RakoBridge
```

Comment:
I think `light` importing bridge is fine, let's guard it only in the opposite direction.